plugins/leemons-plugin-leebrary/frontend/src/request/hooks/keys/simpleAssetList.js

```diff
@@ -0,0 +1,14 @@
+export const allSimpleAssetListKey = [
+  {
+    plugin: 'plugin.leebrary',
+    scope: 'simpleAssetList',
+  },
+];
+
+export const getSimpleAssetListKey = (query) => [
+  {
+    ...allSimpleAssetListKey[0],
+    action: 'get',
+    query,
```

Comment:
It's better if you destructure it, so it can be more easily globally invalidated